### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by botanize**
Basic template with developer's certificate of origin v1.1

Re #29, #27

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Before submitting a pull request, please read
```
Comment (e-lo) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Things that are instructions to the PR Creator can be made in

Comment (botanize) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: I think most of your comment got cut-off

**Review by e-lo — CHANGES_REQUESTED** [review]
See inline comments.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+[CONTRIBUTING.md](CONTRIBUTING.md).
+
+For spec changes:
+1. Update documentation if relevant.
+1. Reference related issues.
+
+Developer's Certificate of Origin 1.1
```
[the remark this answers is not in the document]
Comment (jlstpaul) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: This same text is in CONTRIBUTING.md.  Recommend this language in place of everything after Line 7:

By contributing to this project, all contributors certify to the Developer Certificate of Origin in [CONTRIBUTING.md](CONTRIBUTING.md#contributor-agreement).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+By making a contribution to this project, I certify that:
```
[the remark this answers is not in the document]
Comment (jlstpaul) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: Agree. See above comment to refer to CONTRIBUTING.md#contributor-agreement

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+Before submitting a pull request, please read
+[CONTRIBUTING.md](CONTRIBUTING.md).
+
+For spec changes:
+1. Update documentation if relevant.
+1. Reference related issues.
+
```
[the remark this answers is not in the document]
Comment (botanize) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: We could require that a PR addresses an issue, and use the issue for fact-finding, censusing building, and deciding. If we choose that direction I'll update the PR template to require at least one issue reference.

[the remark this answers is not in the document]
Comment (e-lo) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: ...and can you update the issue template for feature requests to add these steps?

[the remark this answers is not in the document]
Comment (botanize) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: see commit 2d6087c

**Comment by e-lo** [comment]
LGTM!

**Review by e-lo — APPROVED** [review]

**Pull request merged**